client/signup/steps/site-or-domain/index.jsx

```diff
@@ -223,9 +222,8 @@ class SiteOrDomain extends Component {
 		} else if ( designType === 'existing-site' ) {
```

Comment:
Clicking on "Choose site" and then picking any existing site will lead to an infinitely hanging processing screen. I didn't have enough time to dig deeper, but my intuition is that we might need to submit some dependencies here 🤔